app/routes/users/components/StudentConfirmation.tsx

```diff
@@ -1,206 +1,229 @@
+            Abakus tilgang til dine studier og fag i StudentWeb gjennom FEIDE
+            slik at vi kan registrere deg som medlem.
+          </p>
+        )}
+
+        {isStudent != null &&
+          (isStudent ? (
+            <Card info>
+              <h4>Du er verifisert som student</h4>
+              <p className={styles.infoText}>
+                Dersom du ønsker å endre trinn eller studie, vennligst send en
+                forespørsel til{' '}
+                <a href="mailto:webkom@abakus.no">webkom@abakus.no</a> med
+                dokumentasjon på ditt trinn og studie. Gyldig dokumentasjon kan
+                være:
+                <ul className={styles.programmeList}>
+                  <li>
+                    Screenshot eller dokumentasjon fra studentWeb (at du tar fag
+                    tilhørende ditt ønsket trinn er nok)
+                  </li>
+                  <li>Dokumentasjon fra institutt/fakultet</li>
+                </ul>
+              </p>
+            </Card>
+          ) : (
+            <Card danger>
+              <h4>
+                Informasjon vi har hentet om deg fra StudentWeb gir ikke
+                mulighet for automatisk medlemskap i Abakus
+              </h4>
+              <NotEligibleInfo />
+            </Card>
+          ))}
+
+        {authRes && authRes.status !== 'success' && (
+          <Card danger={authRes.status !== 'success'}>
+            {authRes.status === 'unauthorized' && (
+              <>
+                <h4>
+                  Ingen av dine nåværende studier tillater medlemskap i Abakus:
+                </h4>
+                <ul className={styles.programmeList}>
+                  {authRes.studyProgrammes.map((p) => (
+                    <li key={p}>{p}</li>
+                  ))}
+                </ul>
+                <NotEligibleInfo />
+              </>
+            )}
+            {authRes.status === 'error' && (
+              <>
+                <h4>En feil oppsto under validering av din studentstatus:</h4>
+                <p>{authRes.detail}</p>
+              </>
+            )}
+          </Card>
+        )}
+
+        <Button success onClick={() => performStudentAuth()}>
+          Verifiser med FEIDE
+        </Button>
+        {isStudent != null && (
+          <p className={styles.infoText}>
+            Du har allerede verifisert din status. Dersom du har byttet studie
+            og ønsker å bli medlem av Abakus, kan du verifisere deg på nytt og
+            vi vil oppdatere statusen din dersom du er registrert riktig på
+            StudentWeb.
+          </p>
+        )}
+
+        <Modal
+          show={showMemberModal}
+          contentClassName={styles.membershipModalContent}
+          onHide={() => setShowMemberModal(false)}
+        >
+          <Card info>
+            <Flex column>
+              <h4>Din studentstatus ble godkjent!</h4>
+              <p>
+                Du har blitt satt i gruppen: <b>{authRes?.grade}</b>
```

Comment:
```suggestion
                Du har blitt satt i gruppen <b>{authRes?.grade}</b>
```
No need for the colon. It feels more human without imo